texastribune/donations
# 💅🏼 Home Page Copy Updates

#### What's this PR do?
- Updates the main tagline, and the social share image and description.

| Before | After |
| ----- | ----- |
| ![Screenshot 2023-04-13 at 12 43 41 PM](https://user-images.githubusercontent.com/4173104/231841558-d9c42ba4-c758-491e-8d04-6648d9d4bf9b.png) |  ![Screenshot 2023-04-13 at 12 43 50 PM](https://user-images.githubusercontent.com/4173104/231841585-70d88308-d283-4b80-8ae4-c8987c3c1172.png) |

| Before | After |
| ----- | ----- |
| ![social](https://user-images.githubusercontent.com/4173104/231841929-7d72a348-093e-4a47-a9bc-b0098817f56c.png) | ![social](https://user-images.githubusercontent.com/4173104/231841743-412fe8ed-ff7a-4def-8652-f62b180408d5.png) |

| Before | After |
| ----- | ----- |
| At The Texas Tribune, members make public service journalism possible. Become a member today and donate to support the Tribune's nonprofit newsroom. | Members make our journalism possible. Support The Texas Tribune with a donation today. |

#### Why are we doing this? How does it help us?
- #marketing

#### How should this be manually tested?
- Confirm tagline looks good.
- Confirm social share image/description was correctly applied.

#### How should this change be communicated to end users?
- Let Kassie know.

#### Are there any smells or added technical debt to note?
- Nah

#### What are the relevant tickets?
- https://airtable.com/appyo1zuQd8f4hBVx/tbloNZu8GkM52NKFR/viwULM503frwBhAOe/recXuHJvODwdi84XQ?blocks=hide
